docs/app/docusaurus.config.js

```diff
@@ -1,185 +1,197 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-import { themes } from 'prism-react-renderer';
+import { themes } from "prism-react-renderer";
 
 const codeTheme = { light: themes.github, dark: themes.dracula };
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-    title: 'Devbox',
-    tagline: 'Instant, easy, and predictable shells and containers',
-    url: 'https://www.jetify.com',
-    baseUrl: '/devbox/docs/',
-    onBrokenLinks: 'throw',
-    onBrokenMarkdownLinks: 'warn',
-    favicon: 'img/favicon.ico',
-    trailingSlash: true,
+  title: "Jetify Docs",
+  tagline: "Instant, easy, and predictable shells and containers",
+  url: "https://www.jetify.com",
+  baseUrl: "/",
```

Comment:
Wait, I don't think this is right. What are we trying to do here?

Reply:
We're trying to split the docs into Devbox and Cloud, so we can have urls like: `https://www.jetify.com/devbox/docs` and `https://www.jetify.com/cloud/docs`. Further down you'll see that each of the docs folders have their own route base paths: 

```json
 docs: {
          routeBasePath: "/devbox/docs",
          sidebarPath: require.resolve("./sidebars.js"),
          // Please change this to your repo.
          // Remove this to remove the "edit this page" links.
          docItemComponent: "@theme/ApiItem",
          editUrl: "https://github.com/jetify-com/devbox/tree/main/docs/app/",
        },
```

Is there a better way to do this? It works well in Vercel, but not sure if it will interfere with other pages

Reply:
oh I see 👍